test/Interfaces/LazySet.jl

```diff
@@ -1,6 +0,0 @@
-for ST in LazySets.subtypes(LazySet, true)
-    # check that certain methods are implemented
-    isconvextype(ST)
```

Comment:
aren't these nice to keep?

Reply:
I would say not, because they only test that the method does not crash. Since there are default implementations for `LazySet`, these are rather useless. I would add proper tests checking that the result is correct. But this has to be done individually for each set type.